python/ray/tune/logger/aim.py

```diff
@@ -0,0 +1,220 @@
+import logging
+from pathlib import Path
+
+import numpy as np
+from typing import TYPE_CHECKING, Dict, Optional, List
+
+from ray.tune.logger.logger import LoggerCallback
+from ray.tune.result import (
+    TRAINING_ITERATION,
+    TIME_TOTAL_S,
+    TIMESTEPS_TOTAL,
+)
+from ray.tune.utils import flatten_dict
+from ray.util.annotations import PublicAPI
+
+if TYPE_CHECKING:
+    from ray.tune.experiment.trial import Trial  # noqa: F401
+try:
+    from aim.ext.resource import DEFAULT_SYSTEM_TRACKING_INT
+    from aim.sdk import Run
+except ImportError:
+    DEFAULT_SYSTEM_TRACKING_INT = None
+    Run = None
+
+logger = logging.getLogger(__name__)
+
+VALID_SUMMARY_TYPES = [int, float, np.float32, np.float64, np.int32, np.int64]
+
+
+@PublicAPI
+class AimCallback(LoggerCallback):
```

Comment:
To be consistent with the other similar callbacks, we should name it `AimLoggerCallback`.

```suggestion
class AimLoggerCallback(LoggerCallback):
```